bin/reindex

```diff
@@ -29,11 +30,29 @@ echo "Importing all API content"
 echo "Waiting for empty queues"
 ./bin/wait-for-empty-queues
 
+echo "Check for empty index"
+index_check=`.bin/console index:total:write`
+
+if [ "$index_check" -eq "0" ] || [ "$index_check" -eq "1" ]; then
```

Comment:
Just check need to check `[ "$index_check" -eq "0" ]` and we can introduce an additional threshold in a followup.

If we need to do further checks then we should convert to an integer so that we can perform comparisons.

Reply:
Fixed in [last commit](https://github.com/elifesciences/search/pull/406/commits/cc31ce677284be760d58ddea852434e63db49be3)